### ISSUE TITLE
Bash Auto-completion  doesn't work on CL server

### ISSUE THREAD

**Issue by lebensterben** [clearlinux/distribution#2006]
This is first reported here: https://github.com/clearlinux/distribution/issues/1997

Currently bash-completion of `swupd` is always installed, but it relies on a function provided by `bash-completion` package, which is by default installed in CL desktop, but not in CL server.

There are several ways to solve this:
1. Add `bash-completion` to `os-core` bundle
2. Do not install it if `bash-completion` package is not installed

**Comment by lebensterben** [comment]
Specifically, it relies on this script in `bash-completion`
`/usr/share/bash-completion/bash_completion`

**Comment by mdhorn** [comment]
instead of `os-core`, perhaps `sysadmin-basic` which is installed by the installer for both servers and desktop.

**Comment by bwarden** [comment]
`bash-completion` is also part of bundle `shells`, which is itself included by `user-basic`, which also includes `sysadmin-basic`. Not an answer, just more data.

**Comment by mdhorn** [comment]
> `bash-completion` is also part of bundle `shells`, which is itself included by `user-basic`, which also includes `sysadmin-basic`. Not an answer, just more data.

Maybe this changed recently, but `user-basic` is not included in `sysadmin-basic`, but it probably should be.

**Comment by bwarden** [comment]
No... `user-basic` includes `sysadmin-basic`. Don't want to make that circular :)

**Comment by mdhorn** [comment]
> No... `user-basic` includes `sysadmin-basic`. Don't want to make that circular :)

Any idea what `user-basic` pulls in `kvm-host`? That pulls in python and other things, which is a bit more than we want to pull into a server image I think.

Maybe swupd should just pull in `bash-completion` directly?
